render_tests/ui/fail/unclosed-tag-complex.stderr

```diff
@@ -0,0 +1,39 @@
+error: expected one of `)`, `,`, `.`, `?`, or an operator, found `compile_error`
+  --> $DIR/unclosed-tag-complex.rs:12:9
+   |
+11 |         </ul>
+   |             -
+   |             |
+   |             expected one of `)`, `,`, `.`, `?`, or an operator
+   |             help: missing `,`
+12 |       </div>
+   |         ^^^ unexpected token
+
+error: Expected closing tag for: <li>
+  --> $DIR/unclosed-tag-complex.rs:11:11
+   |
+11 |         </ul>
+   |           ^^
```

Comment:
> Continuing to parse after such an error seems undesirable since it's very likely further errors will include nonsensical messages

Yes! makes very much sense 😃